server/db/migrate/20150429180751_add_root_passwords_to_deployments.rb

```diff
@@ -0,0 +1,6 @@
+class AddRootPasswordsToDeployments < ActiveRecord::Migration
+  def change
+    add_column :fusor_deployments, :rhev_root_password, :string
+    add_column :fusor_deployments, :cfme_root_password, :string
```

Comment:
@isratrade, From the discussion, there will be 1 password per product, which does mean that rhev hypervisors and engines will have the same root password initially.  Other options were discussed; however, this is the one that was chosen.